.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,17 +1,16 @@
 ---
```

Comment:
why these files are changed?

Reply:
tbh I have no clue (not manually changed, maybe due to update or yarn?)